### PR TITLE
Add vite debug logs

### PR DESCRIPTION
**Pull request by leeweisberger**
Add vite logs as debug logs so we can tell what's happening on the vite server.

### CODE REVIEW

File: pkg/views/dev.go [hunk 1]
```diff
@@ -242,6 +242,14 @@ func runVite(opts ViteOpts, tmpdir string) (*exec.Cmd, string, error) {
 	if err = cmd.Start(); err != nil {
 		return nil, "", err
 	}
+	logger.Debug("Started vite with process id: %v", cmd.Process.Pid)
+
+	go func() {
```
Comment (calufornia) [inline pkg/views/dev.go:247]: I think you want to start this goroutine after `wg.Wait()` - I tested this locally and the view doesn't load in the editor

Comment (calufornia) [reply pkg/views/dev.go:247]: Also, you probably want to do this only if `!opts.TTY` - in the interactive case, we're already logging debug logs: https://github.com/airplanedev/cli/blob/ac6555d65be1c2a505ba9ba689c59fb50ca05a12/pkg/views/dev.go#L222-L224

Comment (leeweisberger) [reply pkg/views/dev.go:247]: Yeah I misread good catch. Added some comments for posterity

**Comment by leeweisberger** [comment]
I realized this actually fixes the problem.

Talked to @ericliuche and scanner has a max buffer size. Since we are never flushing the buffer, vite crashes after the buffer fills up :hand:

**Comment by kellymtrinh** [comment]
> I realized this actually fixes the problem.
> 
> Talked to @ericliuche and scanner has a max buffer size. Since we are never flushing the buffer, vite crashes after the buffer fills up ✋

Wait does this mean it's not because of erroring/invalid code? (Just that that makes the buffer fill up faster??)

**Comment by leeweisberger** [comment]
> > I realized this actually fixes the problem.
> > Talked to @ericliuche and scanner has a max buffer size. Since we are never flushing the buffer, vite crashes after the buffer fills up ✋
> 
> Wait does this mean it's not because of erroring/invalid code? (Just that that makes the buffer fill up faster??)

Exactly!

**Review by calufornia — APPROVED** [review]

**Pull request merged**